src/services/sync/strategies/TSP/TSP.js

```diff
@@ -0,0 +1,163 @@
+const rp = require('request-promise-native');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'request-promise-native'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)